### ISSUE TITLE
Programação

### ISSUE THREAD

**Issue by LoreleiGab** [hacklabr/viradacultural#128]
A página de programação não carrega os eventos mesmo após rodar o json.

**Comment by leogermani** [comment]
oi Lorelei

Qual é a instalação do Mapas da onde vc está puxando os eventos?

**Comment by LoreleiGab** [comment]
Do branch Virada Cultural 2015 (a mesma que a equipe da SMC utilizou em 2016). Alteramos o getJson2015.php para getJson2017.php onde alteramos o project_id e as datas, porem ele dá o seguinte erro: PHP Notice:  Trying to get property of non-object in /var/www/html/jsons/getJsons2017.php

A linha do erro contem a seguinte informação:
$e->spaceId = $occ->$rule->space("{id,name,singleUrl,shortDescription}");

**Comment by LoreleiGab** [comment]
Perdão... estava testando... a linha é essa
$e->spaceId = $occ->space->id;

**Comment by leogermani** [comment]
Mas já respondo aqui: 

Este tema não é o mais apropriado para um uso para qualquer finalidade. Ele foi desenvolvido para um caso específico e vai requerer customização de código para fazê-lo funcionar. Se quiser algo mais fácil de botar pra rodar veja o https://github.com/hacklabr/cultural

Se quiser tentar arrumar:

Houveram algumas mudanças mesmo na API e o tema precisa ser atualizado.

Olha este commit, veja a mudança feita no arquivo mapasculturais2post.php:

https://github.com/hacklabr/cultural/commit/4439d44faa17e595d5a0c08c90873a7b56cff685

Acho que é basicamente o que você deve precisar mudar. Basicamente, antes as propriedades das ocorrências já vinham por padrão na resposta da API, e agora precisam ser pedidos exatamente quais você quer.

Outra opção, menos recomendada é passar o parâmetro @version = 1 , e aí ele acessa a versão antiga da API (muuuuuito mais lenta)

Veja se consegue seguir por aí

abraços

**Comment by leogermani** [comment]
Deduzi pelas suas respostas que você está falando do Mapas Culturais da cidade de São Paulo, certo? Era essa minha pergunta original.

abraços

**Comment by LoreleiGab** [comment]
Sim, estamos tentando subir o site da Virada Cultural de São Paulo com a marca de vocês. Obrigada pela ajuda.

**Comment by LoreleiGab** [comment]
Pode me indicar, por gentileza, onde passo o o parâmetro @version = 1?

**Comment by leogermani** [comment]
Nas linhas que montam a URL de consulta na API. tipo essas: https://github.com/hacklabr/viradacultural/blob/master/jsons/getJsons.php#L14-L15
